shaders/mesh_shader_culling/mesh_shader_culling.mesh

```diff
@@ -68,10 +69,10 @@ void main()
     if ( do_emit )
     {
         // Vertices:
-        gl_MeshVerticesEXT[0].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_0, 0.0f, 1.0f) + sharedData.positionTransformation + displacement;
-        gl_MeshVerticesEXT[1].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_1, 0.0f, 1.0f) + sharedData.positionTransformation + displacement;
-        gl_MeshVerticesEXT[2].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_2, 0.0f, 1.0f) + sharedData.positionTransformation + displacement;
-        gl_MeshVerticesEXT[3].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_3, 0.0f, 1.0f) + sharedData.positionTransformation + displacement;
+        gl_MeshVerticesEXT[0].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_0, 0.0f, 1.0f) + posTrans + displacement;
+        gl_MeshVerticesEXT[1].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_1, 0.0f, 1.0f) + posTrans + displacement;
+        gl_MeshVerticesEXT[2].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_2, 0.0f, 1.0f) + posTrans + displacement;
+        gl_MeshVerticesEXT[3].gl_Position = vec4(2.0 * sharedData.subDimension * unitVertex_3, 0.0f, 1.0f) + posTrans + displacement;
```

Comment:
Here, `posTrans + displacement` is the above calculated `position`.
Moreover, due to the w-component, `displacement` is a point while `posTrans` is a vector. That is, you're adding two points and a vector which leads to, well, something strange, with a w-component of 2.0f. You should not do that.
If you would just use a `vec2 position`, you could calculate
`vec4(2.0f * sharedData.subDimension * unitVertex_N + position, 0.0f, 1.0f);`